pom.xml

```diff
@@ -28,7 +28,7 @@
     <groupId>org.eclipse.microprofile</groupId>
     <artifactId>microprofile</artifactId>
     <packaging>pom</packaging>
-    <version>2.1-SNAPSHOT</version>
+    <version>2.1-RC1</version>
 
```

Comment:
The Jenkins release builds take care of these type of changes.  Leave this with -SNAPSHOT.